docs/source/tutorial/manage-access-control-list.txt

```diff
@@ -0,0 +1,187 @@
+===========================
+Manage Access to Data (ACL)
+===========================
+
+Access Control List
+===================
+
+A :term:`Resource`'s Access Control Lists (ACLs) determine the access
+to that Resource. An ACL entry can specify following :term:`Access
+Modes` :
+
+- :term:`Read Access`,
+
+- :term:`Append Access`,
+
+- :term:`Write Access`, and
+
+- :term:`Control Access`.
+
+Access (i.e. an ACL entry) can be granted to individual :term:`agents
+<Agent>`, to groups, or even to everyone.
+
+For a Resource, you can define:
+
+- a Resource-specific ACL that applies only to that Resource, and
+
+- an inheritable ACL that applies to the Resource's children as a
+  default/fallback ACL if they don't specify their own
+  Resource-specific ACL.
+
+That is, if a Resource has a Resource-specific ACL, that ACL applies to
+that Resource and only to that Resource. However, if the
+Resource-specific ACL does not exist, the Resource's
+:term:`Container`'s inheritable ACL (or if that is unset, then that of
+its Container's Container, etc.) acts as the Resource's fallback ACL.
+As such, the ACL for a Resource may be defined in separate Resource,
+and retrieving a Resource with its ACL may result in extra HTTP
+requests being sent.
+
+.. note::
+
+   An application can access a Resource's or Container's ACL only if an
+   Authenticated User with :term:`Control Access` to
+   the applicable Resource or Container has authorized the application
+   to manage access on their behalf.
```

Comment:
wasn't sure why we're calling out the applications -- wouldn't it just be another entry in the ACL?

Reply:
Currently, there is a double authorization mechanism: one for users, and one for apps. The idea is that you might want to control how people access your data, and thus constrain the apps which are used by the people you gave access to. For instance, you might want to give access to your photos to your relatives, but not if they want to access them with an app you don't trust.

Important changes in this mechanism are actively being discussed at the spec level, as it is not ideal, but that is currently what is enforced by Solid servers.

Reply:
Ah, thanks you two!  Definitely clears up why we were calling that out.  Made into a topic instead of a note then.
https://kay.inrupt.net/public/docs-staging/solid-client-js/tutorial/manage-access-control-list.html

Will close this pull request since I was able to create a branch to this repo 😄  Will create a pull request from that branch instead.